backend/src/main/java/codezap/member/domain/Member.java

```diff
@@ -1,18 +1,25 @@
 package codezap.member.domain;
 
+import java.util.Objects;
+
 import jakarta.persistence.Column;
 import jakarta.persistence.Entity;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
 
+import codezap.global.auditing.BaseTimeEntity;
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 @Entity
-@Getter
 @NoArgsConstructor
```

Comment:
⚡
```suggestion
@NoArgsConstructor(access = AccessLevel.PROTECTED)
```

@NoArgsConstructor의 기본 접근제어자를 찾아보니 public으로 설정 되어있네요. protected로 접근제어자를 좁히면 더 좋을 것 같습니다.